engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActorData.scala

```diff
@@ -22,7 +20,8 @@ final case class WorkflowExecutionDiff(executionStore: Map[JobKey, ExecutionStat
 case class WorkflowExecutionActorData(workflowDescriptor: EngineWorkflowDescriptor,
                                       executionStore: ExecutionStore,
                                       backendJobExecutionActors: Map[JobKey, ActorRef],
-                                      outputStore: OutputStore) extends WdlLookup {
+                                      outputStore: OutputStore,
+                                      restarting: Boolean) extends WdlLookup {
```

Comment:
Hmm, presumably this is known at WorkflowExecutionActor construction time? Can we make it a props argument instead to avoid the message receiving / state flipping dance?